ES/ESmy001.xml

```diff
@@ -30,324 +30,712 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                           ዓለም፡ አሜን፨ <hi rend="rubric">ወኮነ፡ ሶበ፡ ወፅኡ፡ ወተካፈሉ፡ </hi>ሐዋርያት፡ ላዕለ፡ ደብረ፡ ዘይት፡ ወእንዘ፡ ይነብቡ፡ በበይናቲሁሙ፡ ስብሓተ፡ እግዚአብሔር፨
+                            ወበእንተዝ፡ አስተርአየ፡ ሎሙ፡ እግዚአነ፡ ወድኀኒነ፡
+                        </incipit>
+                     </msItem>
+
+                     <msItem xml:id="ms_i1.19">
+                        <locus from="61ra" to="63va"/>
+                        <title type="complete" ref="LIT3536Martyr">The Martyrdom of St Philip</title>
+                        <textLang mainLang="gez"/>
+                        <incipit xml:lang="gez"><hi rend="rubric">በስመ፡ አብ፡ ወወልድ፡</hi> ወመንፈስ፡ ቅዱስ፡ ፩
+                          እግ<hi rend="rubric">ዚአብሔር፡ ዝንቱ፡ ስምዑ፡</hi> ለሐዋርያ፡ ቡሩክ፡ ፊልጶ<hi rend="rubric">ስ፡ ዘተጋደለ፡ <sic resp="DN">ገድል፡</sic></hi>
+                          ሠናየ፡ አመ፡ <hi rend="rubric">፲</hi>ወ<hi rend="rubric">፰</hi>ለወርኀ፡ ሕዳር፡ በሰላመ፡ እግዚአብሔር፡ አሜን፨፨ <hi rend="rubric">ወኮነ፡ ሶበ፡ ወፅአ፡ ቡሩክ፡</hi> ፊልጶስ፡ ውስተ፡
+                          ሀገረ፡ አፍራቂያ፡ ኀበ፡ ሰብአ፡ ይእቲ፡ ብሔር፡ ወሰበከ፡ ሎሙ፡ በሐዲስ፡ አምላክ፡ ዘኢያአምርዎ፡ ስሞ፡ ኢየ<cb n="b"/>ሱስ፨
+                        </incipit>
+                     </msItem>
+
+                     <msItem xml:id="ms_i1.20">
+                        <locus from="63vb" to="72vb"/>
+                        <title type="complete" ref="LIT2182Preach">The Preaching of St Andrew</title>
+                        <textLang mainLang="gez"/>
+                        <incipit xml:lang="gez"><hi rend="rubric">በስመ፡ አብ፡ ወወልድ፡</hi> ወመንፈስ፡ ቅዱስ፡ አሐ<hi rend="rubric">ዱ፡ እግዚአብሔር፡ ስብ</hi>ከቱ፡
+                          ለብፁዕ፡ ረድእ፡ እ<hi rend="rubric">ንድርያስ፡ ሐዋርያሁ፡</hi> ለኢየሱስ፡ ክርስቶስ፡ ወዜናሁ፡ ዘኮነ፡ ውስተ፡ ሀገረ፡ እክናድ፡ ወሀገረ፡ ልድያ፡
+                          በሰላመ፡ እግዚእነ፡ ኢየሱስ፡ ክርስቶስ፡ ለዓለም፡ አሜን፨ ወእንዘ፡ ይወፅኡ፡ አርድእት፡ ውስተ፡ ዓለም፡ ይስብኩ፡ በዜና፡ መንግሥተ፡
+                          ሰማያት፡ አስተርአየ፡ ሎሙ፡ እግዚእነ፡ ወተናገሩ፡ ከመዝ፡
+                        </incipit>
+                     </msItem>
+
+                     <msItem xml:id="ms_i1.21">
+                        <locus from="72vb" to="89ra"/>
+                        <title type="complete" ref="LIT3537Acts">The Martyrdom of Sts Andrew and Bartholomew</title>
+                        <textLang mainLang="gez"/>
+                        <incipit xml:lang="gez">
+                          <hi rend="rubric">በስመ፡ አብ፡ ወወልድ፡</hi> ወመንፈስ፡ ቅዱስ፡ <hi rend="rubric">ዝንቱ፡ ግብሮሙ፡</hi> ለክልኤቱ፡
+                          አርድእት፡ ቡ<hi rend="rubric">ሩካን፡ እንድርያስ፡</hi><pb n="73ra"/> ወበርተሎሜዎስ፡ ሐዋርያቲሁ፡
+                          ለእግዚእነ፡ ኢየሱስ፡ ክርስቶስ፡ እለ፡ ገብሩ፡ ውስተ፡ ሀገረ፡ በርቶስ፡ ወእምድኅረ፡ ግብአቶሙ፡
+                          እምሀገረ፡ ሳዎ፡ ዘይብልዋ፡ አልዋኅ፡ በሰላመ፡ እግዚእነ፡ ኢየሱስ፡ ክርስቶስ፡ ለዓለም፡ አሜን፨ ወኮነ፡
+                          እምድኅረ፡ ተንሥኦቱ፡ ለእግዚእነ፡ ኢየሱስ፡ ክርስቶስ፡ እማእከለ፡ እሙታን፡ ወንጉሥ፡
+                          ውእቱ ፡ ላዕለ፡ ሰማይ፡ ወምድር፡ ወአስተርአዮ፡ ለበርተሎሜዎስ፡ በውስተ፡ ሀገረ፡
+                          አሕዛብ፡ ውስተ፡ በሓውርተ፡ አቅጥራን፡ ዘይእቲ፡ ሀገረ፡ አዝርዮስ፨
+                        </incipit>
+                        </msItem>
+
```

Comment:
wrong nesting, delete the second closing /msItem tag

Reply:
ok